src/build/dotnet/steps/nuget-package.steps.yaml

```diff
@@ -13,25 +13,61 @@ steps:
       skip: ${{ parameters.skipDependencyChecks }}
       continueIfVulnerabilities: true
 
-  - template: /src/build/dotnet/tasks/netcore/version.yaml
-    parameters:
-      releaseBranchName: ${{parameters.releaseBranchName}}
-      packageSource: ${{parameters.packageSource}}
+  - ${{ if eq(parameters.applyVersioning, true) }}:
```

Comment:
`applyVersioning` is missing from the parameters

Reply:
Please can you review further missing parameters in all files? This is because the task files had parameters that had defaults are were not already passed from the job/steps files

Reply:
I've amended params, I believe everything should be there now but please correct me if its not the case